src/components/interface/PendingTransaction/PendingTransaction.tsx

```diff
@@ -62,7 +62,7 @@ const PendingTransaction: React.FunctionComponent<PendingTransactionProps> = ({
   const cachedMargin = useRef<number | undefined>(margin);
 
   useEffect(() => {
-    if (!isUndefined(margin) && margin !== 0){
+    if (!isUndefined(margin) && margin !== 0 && fetch == 0){
       cachedMargin.current =  margin;
     }
   }, [margin]);
```

Comment:
are you missing fetch in the dependency list?

Reply:
the aim here is to cache the margin only when the component is initialised (& margin has a value). I don't want it to reload with `fetch`

Reply:
LGTM